src/gui/components/wxTitleManagerList.cpp

```diff
@@ -935,6 +941,14 @@ wxString wxTitleManagerList::GetTitleEntryText(const TitleEntry& entry, ItemColu
 		return "";
 		//return wxStringFormat2("{}", entry.format);
 	}
+	case ColumnLocation:
+	{
+		if (_pathToUtf8(entry.path).find("usr/") != std::string::npos ||
+			_pathToUtf8(entry.path).find("sys/") != std::string::npos)
+			return _("MLC01");
```

Comment:
In hindsight, I think using MLC instead of MLC01 is slightly better since Cemu's general settings also call it the MLC path and not MLC01 path.
```suggestion
			return _("MLC");
```